config/default.js

```diff
@@ -1,11 +1,14 @@
+const PostGraphileConnectionFilterPlugin = require('postgraphile-plugin-connection-filter');
+// const path = require('path');
+
 module.exports = {
   app: {
     port: process.env.PORT || 3000,
     suppressErrorStack: process.env.SUPPRESS_ERROR_STACK || false,
   },
   db: {
-    user: '',
-    host: 'db',
+    user: 'plyo_boilerplate',
+    host: 'localhost',
```

Comment:
```suggestion
    host: 'db',
```

We usually run it in docker locally, localhost will not be resolved correctly